deploy/examples/create-external-cluster-resources.py

```diff
@@ -826,7 +938,42 @@ def validate_rados_namespace(self):
             raise ExecutionFailureException(
                 ("The provided rados Namespace, '{}', is not found in the pool '{}'").format(
                     rados_namespace, rbd_pool_name))
-
+    
+    def get_rgw_fsid(self):
+        access_key = self.out_map['RGW_ADMIN_OPS_USER_ACCESS_KEY']
+        secret_key = self.out_map['RGW_ADMIN_OPS_USER_SECRET_KEY']
+        rgw_endpoint = self._arg_parser.rgw_endpoint
+        base_url =  self.endpoint_dial(rgw_endpoint) + "://"
+        cert = None
+        if self._arg_parser.rgw_tls_cert_path:
+            cert = self._arg_parser.rgw_tls_cert_path
+        base_url = base_url + rgw_endpoint + "/admin/info?"   
+        params = {'format': 'json'}
+        request_url = base_url + urllib.parse.urlencode(params)
+            
+        try:
+            r = requests.get(request_url, auth=S3Auth(access_key, secret_key,rgw_endpoint), cert=cert)
```

Comment:
Can `cert` be empty in `get()`? Is there any option to do insecure TLS?

Reply:
Yes it can, added a check for `self.arg_parser.rgw_skip_tls == False` 

https://github.com/rook/rook/pull/10006#discussion_r861489844 can you plz give reply to it so I can verify too

Reply:
When `self.arg_parser.rgw_skip_tls == True` we **must** pass an option like insecure TLS.

Reply:
these are the get parameter we can apply https://www.w3schools.com/python/ref_requests_get.asp,

In which we can apply `verify=false`

Reply:
Yes, that's what we want when `rgw_skip_tls` is True.